### DISCUSSION TITLE
How to disable placeholder?

### DISCUSSION THREAD

**Discussion by AgataJedryszek-da** [JedWatson/react-select#5973] [Q&A]
I don't want a placeholder in my implementation, but when i set `placeholder={undefined}` it shows a default placeholder `"Select..."`. Can I disabled it without using Placeholder component?

**Comment by manjushsh** [comment]
Hi @AgataJedryszek-da, If I understand correctly, You don't want to show default placeholder. You can hide it by passing empty placeholder as [shown here in the sandbox](https://codesandbox.io/p/sandbox/codesandboxer-example-forked-rlj397)

Code:
```tsx
<Select
        options={colourOptions}
        placeholder={""}
      />
```

**Reply by AgataJedryszek-da** [reply]
Hi, thanks for the reply @manjushsh ! I have done that, but it still returns an empty `<div>`:
![image](https://github.com/user-attachments/assets/850648d6-0093-44a3-9f07-25e443b9ea1e)

I was wondering if there is a way to not render an unnecessary `<div>`.

**Reply by manjushsh** [reply]
As per Select component code https://github.com/JedWatson/react-select/blob/5f93c76c9f8be8ad40a8d7b2bb080b735e1f95d0/packages/react-select/src/Select.tsx#L1787C3-L1787C27 it will render if no value is provided.
